sdk/tests/test_restrict_transactions_between_effective_dates.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+import pytz
+import finbourne_access
+import json
+
+from datetime import datetime, timedelta
+from finbourne_access import api as sa
+from finbourne_access import models as access_models
+from finbourne_access.utilities import ApiClientFactory
+
+
+class FinbourneAccessTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        api_client = ApiClientFactory()
+        cls.policies_api = api_client.build(sa.PoliciesApi)
+
+    def test_restrict_transactions_between_effective_dates(self):
+
+        """
+        This policy allows you to restrict the creation, deletion or updating of
+         transactions between two effective dates (in this case 1 May 2022 to 1 June 2022).
+
+        We specify the effective date restriction in the 'for' attribute of the PolicyCreationRequest
+        model.
+        """
+
+        scope = "ibor"
+
+        policy_code = f"restrict-transactions-on-scope-{scope}-for-may-2022"
+
+        deactivate = datetime(9999, 12, 31, tzinfo=pytz.utc)
+
+        for_spec = access_models.ForSpec(
+            effective_range=access_models.EffectiveRange(
+                _from=datetime(2022, 5, 1, tzinfo=pytz.utc).isoformat(),
+                to=datetime(2022, 6, 1, tzinfo=pytz.utc).isoformat(),
+            )
+        )
+
+        when_spec = access_models.WhenSpec(
+            activate=datetime.now(tz=pytz.utc) - timedelta(days=1), deactivate=deactivate,
+        )
+
+        restrict_selector = access_models.SelectorDefinition(
+            id_selector_definition=access_models.IdSelectorDefinition(
+                identifier={"scope": scope, "code": "1250"},

```

Comment:
what its scope and code here? what is the significance of "1250"? add some commentary

Reply:
* They are just arbitrary scopes and codes for demonstration. I have added comments to make this clearer.